charts/timescaledb-multinode/templates/statefulset-timescaledb-accessnode.yaml

```diff
@@ -107,55 +107,50 @@ spec:
         - containerPort: 5432
         volumeMounts:
         - name: storage-volume
-          mountPath: "{{ .Values.persistentVolume.mountPath }}"
-          subPath: "{{ .Values.persistentVolume.subPath }}"
+          mountPath: "{{ .Values.accessNode.postgresql.persistentVolume.mountPath }}"
+          subPath: "{{ .Values.accessNode.postgresql.persistentVolume.subPath }}"
         resources:
-{{ toYaml .Values.resources | indent 10 }}
-    {{- with .Values.nodeSelector }}
+{{ toYaml .Values.accessNode.postgresql.resources | indent 10 }}
+    {{- with .Values.accessNode.nodeSelector }}
       nodeSelector:
 {{ toYaml . | indent 8 }}
     {{- end }}
-    {{- with .Values.tolerations }}
+    {{- with .Values.accessNode.tolerations }}
       tolerations:
 {{ toYaml . | indent 8 }}
     {{- end }}
     {{- if .Values.schedulerName }}
       schedulerName: {{ .Values.schedulerName }}
     {{- end }}
-    {{- if .Values.affinity }}
```

Comment:
not sure if this `if` can be dropped, and even if it can maybe better leaving it here with the motivation of not making changes where they're not required

Reply:
> better leaving it here with the motivation of not making changes where they're not required

Will revert.